plugins/cisco_meraki_logs.yaml

```diff
@@ -0,0 +1,316 @@
+version: 0.0.1
+title: Cisco Meraki
+description: Log parser for Cisco Meraki
+parameters:
+  - name: listen_port
+    type: int
+    default: 5140
+  - name: listen_address
+    type: string
+    default: "0.0.0.0"
+
+template: |
+  receivers:
+    udplog:
+      listen_address: "{{ .listen_address }}:{{ .listen_port }}"
+      add_attributes: true
+      operators:
+        - type: router
+          default: catch_all_parser
+          routes:
+            - output: meraki_parser
+              expr: 'body matches "^<[^>]+>[\\d]+\\s*[\\d\\.]*\\s*[^\\s]*\\s*[^\\s]*\\s*[\\s\\S]*"'
+        - id: meraki_parser
+          type: regex_parser
+          regex: '^<(?P<priority>[^>]+)>(?P<version>[\d]+)\s*(?P<timestamp>[\d\.]*)\s*(?P<hostname>[^\s]*)\s*(?P<app_name>[^\s]*)\s*(?P<message>[\s\S]*)'
+          timestamp:
+            parse_from: attributes.timestamp
+            layout_type: epoch
+            layout: "s.ns"
+          output: severity_parser
+        
+        - id: catch_all_parser
+          if: 'body matches "^<[\\d]+>"'
+          type: regex_parser
+          regex: '^<(?P<priority>[^>]+)>(\s*)?(?P<message>[\s\S]*)'
+          output: severity_parser
+
+        - type: severity_parser
+          if: "attributes.priority != nil"
+          parse_from: attributes.priority
+          output: message_router
+          mapping:
+            fatal2:
+              [
+                0,
+                8,
+                16,
+                24,
+                32,
+                40,
+                48,
+                56,
+                64,
+                72,
+                80,
+                88,
+                96,
+                104,
+                112,
+                120,
+                128,
+                136,
+                144,
+                152,
+                160,
+                168,
+                176,
+                184,
+              ]
```

Comment:
Is this an auto-format thing? Nothing wrong with it, but I think other plugins has these syslog severity lists on a single line. I think we do that just for the sake of brevity.

Reply:
This was a copy paste from the other plugin. I can change it to a single line for better readability.

Reply:
Updated